Assets/_BForBoss/_Core/Scripts/Environment/PlayerTriggerAreaEffect.cs

```diff
@@ -0,0 +1,142 @@
+using System;
+using Perigon.Character;
+using Perigon.Entities;
+using Perigon.Utility;
+using Sirenix.OdinInspector;
+using UnityEngine;
+
+namespace BForBoss
+{
+    [RequireComponent(typeof(Collider))]
+    public class PlayerTriggerAreaEffect : MonoBehaviour
+    {
+        private enum HealthEffect
+        {
+            Damage,
+            Heal
+        }
+
+        private enum SpeedModifier
```

Comment:
I don't think you need these enum's anymore ,since you have the `Serializable` PlayerMutation

Reply:
Do you mean to place these enums within the `PlayerMutation` class?

Reply:
No, to literally just remove them.

You can just setup the mutation struct object to have the correct settings.

Having this complicates things with
2 different enums + struct to wrap your head around, and gets confusing